root.html

```diff
@@ -17,6 +17,9 @@
     <link rel="icon" type="image/png" href="/static/images/favicon/favicon-32x32.png" sizes="32x32">
     <link rel="icon" type="image/png" href="/static/images/favicon/favicon-96x96.png" sizes="96x96">
 
+    <link rel="preload" as="font" href="/fonts/mattermosticons/mattermosticons.eot?16343536" type="font" crossorigin="anonymous">
```

Comment:
I don't understand what this was supposed to do: `/fonts/mattermosticons/mattermosticons.eot?16343536` is effectively a 404, and just routes to `index.html`:

```
> http "https://community.mattermost.com/fonts/mattermosticons/mattermosticons.eot?16343536" | tail
                <div class='round round-3'></div>
            </div>
        </div>
    </div>
    <div id='root-portal'></div>
    <noscript>
        To use Mattermost, please enable JavaScript.
    </noscript>
</body>
</html>
```

@nevyangelova, can you help explain these changes?

Regardless, at this point, I'd advocate for leaving it out of v5.24.